terraform/deployments/govuk-publishing-infrastructure/content_publisher_activestorage_iam.tf

```diff
@@ -0,0 +1,105 @@
+# Replication IAM role/policy
+
+data "aws_iam_policy_document" "content_publisher_activestorage_replication_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["s3.amazonaws.com"]
+    }
+    effect = "Allow"
+  }
+}
+
+resource "aws_iam_role" "content_publisher_activestorage_replication_role" {
+  name               = "govuk-content-publisher-activestorage-replication-role"
+  assume_role_policy = data.aws_iam_policy_document.content_publisher_activestorage_replication_role.json
+}
+
+data "aws_iam_policy_document" "content_publisher_activestorage_replication_policy" {
+  statement {
+    actions = [
+      "s3:GetReplicationConfiguration",
+      "s3:ListBucket"
+    ]
+    resources = [aws_s3_bucket.content_publisher_activestorage.arn]
+    effect    = "Allow"
```

Comment:
optional: `aws_iam_policy_document` lets you drop `effect = "Allow"` for slightly better readability. Might be easier to clean em all up in one go later though so feel free to ignore for now.